packages/react/src/Link/Link.tsx

```diff
@@ -0,0 +1,50 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2021 Gemeente Utrecht
+ * Copyright (c) 2021 Robbert Broersma
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+import { Link as CommunityLink } from '@utrecht/component-library-react'
+import clsx from 'clsx'
+import { AnchorHTMLAttributes, ForwardedRef, forwardRef, PropsWithChildren } from 'react'
+
+export type LinkColor = 'default' | 'black' | 'white'
+
+export type LinkVariant = 'default' | 'bold'
+
+export interface LinkProps extends AnchorHTMLAttributes<HTMLAnchorElement> {
+  href: string
+  placeholder?: string
+  external?: boolean
+  rel?: string
```

Comment:
Not needed to define a prop here. We extend the `AnchorHTMLAttributes<HTMLAnchorElement>` interface which are declared there